ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionLogUpdatesReader.scala

```diff
@@ -34,9 +34,9 @@ object TransactionLogUpdatesReader {
               _,
             )
           ),
-          treeEventWitnesses = raw.treeEventWitnesses,
-          flatEventWitnesses = raw.flatEventWitnesses,
-          submitters = raw.submitters,
+          treeEventWitnesses = raw.treeEventWitnesses.map(Ref.Party.assertFromString),
+          flatEventWitnesses = raw.flatEventWitnesses.map(Ref.Party.assertFromString),
+          submitters = raw.submitters.map(Ref.Party.assertFromString),
```

Comment:
Changes not imposed by the scope of this PR, but I've done it for having a homogeneous domain representation of types in the `TransactionLogUpdate` DTOs.